src/index.ts

```diff
@@ -328,9 +352,9 @@ export default class D2Reader {
       await this.searchModule?.goToSearchID(href, index, current);
     }
   };
-  clearSearch = () => {
+  clearSearch = async () => {
     if (this.navigator.rights?.enableSearch) {
-      this.searchModule?.clearSearch();
+      await this.searchModule?.clearSearch();
```

Comment:
this was just brought over from how it was before, the entire search needs to be reviewed , therefore i didn't want to change too much here, since this is how it's right now in 1.9